python/tutorials/06-fused-attention.py

```diff
@@ -17,6 +17,8 @@
 import triton
 import triton.language as tl
 
+torch_dtype = torch.float8_e5m2fnuz
+TORCH_HAS_FP8 = hasattr(torch, 'float8_e5m2fnuz')
```

Comment:
Do you plan to use this to check whether we need to use the copy function of torch native fp8 to initialize the input?

Reply:
Yes, that is the plan, so it can switch automatically, and works no matter pytorch supports our data types.